sdk/spring/spring-messaging-azure/src/main/java/com/azure/spring/messaging/implementation/annotation/AzureListenerAnnotationBeanPostProcessorAdapter.java

```diff
@@ -194,16 +194,14 @@ public Object postProcessAfterInitialization(Object bean, String beanName) throw
             if (annotatedMethods.isEmpty()) {
                 this.nonAnnotatedClasses.add(targetClass);
                 if (LOGGER.isTraceEnabled()) {
```

Comment:
do we need this `if (LOGGER.isTraceEnabled())`?

Reply:
Yes, not needed